NIDS/train_dns.py

```diff
@@ -0,0 +1,132 @@
+"""
+This file contains code to train an AE-based model for dns.log data using the parameters from the best model.
+Arguments: the dates of the data to be trained on.
+#TODO: change to AE instead of KitNet 
+
+Command: python train_dns.py --log-dir /usr/local/logs
+
+Authors: 
+- Zoe Hsu <wh2405@nyu.edu> 
+- Olive Song <js10417@nyu.edu> 
+- Diego Lopez <dtl310@nyu.edu>
+- Zihang Xia <zx961@nyu.edu>
+"""
+import argparse
+import random
+import pandas as pd
+import numpy as np
+import pysad
+from pysad.models.kitnet import KitNet
+from sklearn.preprocessing import OneHotEncoder, StandardScaler
+from sklearn.pipeline import Pipeline
+import sys
+import logging
+import os 
+#from NIDS.package.data_preprocess import * 
+from joblib import dump, load
+import subprocess
+import json 
+import gzip
+from utils import *
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s (%(filename)s)')
+#MODEL_PATH = 'kit.joblib'
+#LOAD_MODEL = False       
+
+def ungzip(file_path):
```

Comment:
this function already exists in `train.py` so it should not be duplicated, lets move this to `utils.py` and import into both modules